src/components/kpi/kpi.tmpl.html

```diff
@@ -20,11 +20,15 @@
           <h6>{{kpi.name.toUpperCase()}}</h6>
         </div>
         <alerts-config kpi="kpi" button-html="
-          <div class='im-button alert-caption'>
-            <span><a href='' ng-bind='kpi.layout.text.alert' /></span>
+          <div class='im-button alert-config-btn'>
+            <i class='fa fa-bell'></i>
```

Comment:
Yes, @adamaziz15 please move this down in the HTML so it is above the kpi remove button & commented. The styling is absolute so it wont effect the positioning, but the html will read better.